addon/adapters/application.js

```diff
@@ -41,7 +41,14 @@ export default Ember.Object.extend(FetchMixin, Evented, {
   */
   url: Ember.computed('type', {
     get() {
-      const config = this.container.lookupFactory('config:environment');
+      let config;
+
+      if (typeof Ember.getOwner === 'function') {
```

Comment:
@andrewmp1 I'm curious if this could just be a utility function e.g. `factoryFor('type:thing')` then we could import and use the utility in a few places where needed and later replace with the future `factoryFor` that may ship as the final public API for this behavior